protractor/e2e/integration/blog-page.spec.ts

```diff
@@ -0,0 +1,34 @@
+import {$$, browser, $, ExpectedConditions, protractor} from "protractor";
+import {MainPo} from "../support/main.po";
+import {BlogPo} from "../support/blog.po";
+
+describe('tests for Blog page', ()=> {
+    const BlogPage = new BlogPo();
+
+    beforeEach(async ()=>{
+        await BlogPage.navigateTo();
+
+    });
+
+
+    fit('check the redirection to the 2nd page on Blog page',async () => {
```

Comment:
don't forget to remove "fit" from your tests before pushing it to the repository. Just imagine, that you have 1000 tests in your app and now in your CI will be only 1 test ran:-)